WPMedia/fusion-news-theme-blocks
# Bug/asub 8456 reset password login redirect issue

**Be sure to run `npm test`, `npm run lint`, and write detailed test steps before requesting reviewers**

## Description
Fix for Irish News redirect from reset password issue. When selecting redirect to previous page, after login the user would be redirected to the reset password page they were on so it looks like they never reset their password and logged in. This will redirect them to the original redirect path.

#### Jira Ticket: [THEMES-](https://arcpublishing.atlassian.net/browse/THEMES-)
https://arcpublishing.atlassian.net/browse/ASUB-8456
_Information about what you changed for this PR. Include any dependencies, companion PRs, relevant screenshots, or config changes._

## Test Steps
- Add Reset Password URL to the login page. Make sure it is coming from the correct reset password page.
<img width="1494" alt="Screenshot 2024-05-20 at 10 03 45 PM" src="https://github.com/WPMedia/arc-themes-blocks/assets/5377648/970229ae-2584-4dd3-ac2c-69fd8675af3d">
- Make sure you also have a Redirect URL as well.
- Go to your reset password page and reset your password. It will prompt you to login after.
- Click login and then login and it should redirect you to the original Redirect URL and not the previous page you were on.

1. Checkout this branch `git checkout {branch name}`
2. Run fusion repo with linked blocks `npx fusion start -f -l {blocks to link}`
3. {Next ....}
